Description: Add shebang to the git-rename-tag
 This patch add the shebang needed to be executed directly to the
 git-rename-tag file.
 .
 git-extras (1.7.0-1) unstable; urgency=low
 .
   * Initial release (Closes: #632113)
Author: Jesus Espino <jespinog@gmail.com>
Bug-Debian: http://bugs.debian.org/632113
Forwared: yes
Applied-Upstream: commit:465eabef9b9ce58d2e88c125178353afdc51084e

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- git-extras-1.7.0.orig/bin/git-rename-tag
+++ git-extras-1.7.0/bin/git-rename-tag
@@ -1,3 +1,5 @@
+#!/bin/sh
+
 old=$1
 new=$2
 
